routes/defaulthttp/default_handler.go

```diff
@@ -0,0 +1,108 @@
+package defaulthttp
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"path"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+var mp = make(map[uuid.UUID]User)
+
+// User is ...
+type User struct {
+	Name       string  `json:"name"`
+	Age        int     `json:"age"`
+	Percentage float64 `json:"percentage"`
+	Time       string  `json:"time"`
+}
+
+func decodeJSON(rw http.ResponseWriter, r *http.Request) User {
+	var u User
+	err := json.NewDecoder(r.Body).Decode(&u)
+	if err != nil {
+		http.Error(rw, err.Error(), 400)
+		log.Fatal(err)
+	}
+	return u
+}
+
+func writeResponse(rw http.ResponseWriter, s User) {
```

Comment:
Please use this `writeResponse ` function for all your http responses, weather it is success/error and data

Reply:
I thought of doing it but was unable to do it before hand there is type miss-match, like I am sending response as message sometimes and struct sometimes which is converted to JSON. How can I do this, can you help me with this?

Reply:
Check this file, https://github.com/ashokhirpara1/go-jwt-auth/blob/master/routes/responses.go and if you still don't understand then let me know i will explain you.